larq_compute_engine/core/bgemm_kernels_arm.h

```diff
@@ -49,6 +49,30 @@ struct BgemmKernel<ruy::Path::kNeonDotprod, LhsScalar, RhsScalar, DstScalar,
     TFLITE_DCHECK(false);
   }
 };
+#if RUY_PLATFORM_NEON && RUY_OPT(ASM) && RUY_PLATFORM_NEON_32
+// A BGEMM kernel for ARM32 Neon.
+#include "bgemm_kernels_arm32.h"
```

Comment:
To fix the current CI failure I think you need to add this `"bgemm_kernels_arm32.h"` header dependency to the Bazel build file here: https://github.com/larq/compute-engine/blob/38a92d960d9b100ac39909c391fcec5dd870c698/larq_compute_engine/core/BUILD#L96-L99